### PR TITLE
Allow scheduling downlink messages as JSON in the Console

### LINKED ISSUE #3320: Allow scheduling downlink messages as JSON in the Console
#### Summary

Currently, we allow sending only raw bytes 

#### Why do we need this?

1. Match with the v2 console
2. More flexibility

#### What is already there? What do you see now?

Screenshot 2020-10-06 at 10 21 39

#### What is missing? What do you want to see?

Functionality to send JSON payload instead of just raw bytes.

#### How do you propose to implement this?

UI similar to the v2 console:
Screenshot 2020-10-06 at 10 25 35

Send JSON as [decoded_payload](https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.10/api/messages.proto#L245) instead of `frm_payload`.

#### How do you propose to test this?

e2e and manual testing

#### Can you do this yourself and submit a Pull Request?

yes

### PR DESCRIPTION
**Pull request by bafonins**
#### Summary

Closes https://github.com/TheThingsNetwork/lorawan-stack/issues/3320

<details>
<summary>Screenshots</summary>
Screenshot 2021-10-07 at 16 28 49
Screenshot 2021-10-07 at 16 29 22
Screenshot 2021-10-07 at 16 29 54

</details>

#### Changes

- Add `<CodeEditor />` field to downlink messaging form

#### Testing

Manual + existing e2e

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [ ] Documentation: Relevant documentation is added or updated.
- [x] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

**Review by adriansmares — APPROVED** [review]
Great stuff

File: pkg/webui/console/components/downlink-form/downlink-form.js [hunk 1]
```diff
@@ -120,13 +154,34 @@ const DownlinkForm = ({ appId, devId, device, downlinkQueue, skipPayloadCrypto }
...
-          description={m.payloadDescription}
-          name="frm_payload"
-          component={Input}
-          type="byte"
-          unbounded
-        />
+          name="_payload_type"
+          title={m.payloadType}
+          component={RadioButton.Group}
+          onChange={setPayloadType}
```
Comment (kschiffer) [inline pkg/webui/console/components/downlink-form/downlink-form.js:160]:
```suggestion
          onChange={setPayloadType}
          horizontal
```

Comment (kschiffer) [inline pkg/webui/console/components/downlink-form/downlink-form.js:160]:
```suggestion
          onChange={setPayloadType}
          horizontal
```

**Review by kschiffer — APPROVED** [review]
Nice! See my small remark about making the radio buttons horizontal.

**Pull request merged**